source/server/active_raw_udp_listener_config.cc

```diff
@@ -0,0 +1,26 @@
+#include "server/active_raw_udp_listener_config.h"
+
+#include "server/connection_handler_impl.h"
+
+namespace Envoy {
+namespace Server {
+
+Network::ConnectionHandler::ActiveListenerPtr
+ActiveRawUdpListenerFactory::createActiveUdpListener(Network::ConnectionHandler& parent,
+                                                     Network::ListenerConfig& config) const {
+  return std::make_unique<ConnectionHandlerImpl::ActiveUdpListener>(
+      dynamic_cast<ConnectionHandlerImpl&>(parent), config);
+}
+
+Network::ActiveUdpListenerFactoryPtr
+ActiveRawUdpListenerConfigFactory::createActiveUdpListenerFactory(
+    const Protobuf::Message& /*message*/) {
+  return std::make_unique<Server::ActiveRawUdpListenerFactory>();
+}
+
+std::string ActiveRawUdpListenerConfigFactory::name() { return "raw_udp_listener"; }
```

Comment:
I think we could have const std::string values defined in a new class in source/extensions/filters/listener/well_known_names.h
or we could add a new well_known_names.h file

Reply:
AFAIK files under /source/server shouldn't depends on /source/extensions. What is the appropriate location for such well_known_names.h given that it should also define const string for "quic_listener"

Reply:
Yeah we don't really have a good situation here for this. There are certain cases in which we do refer back into extensions for well known names, though in this case I think it's probably fine to just have this be a constant string here as long as we don't need it anywhere else?

Reply:
We use it in connection_handler_impl to initialize default value if not configured.

Reply:
If we use the string in multiple places I would define this in a common file somewhere. This could be inside server or somewhere else?